### ISSUE TITLE
tests should also be done with MPIR

### ISSUE THREAD

**Issue by videlec** [aleaxit/gmpy#158]
There are some subtle differences between GMP and its fork [MPIR](http://mpir.org/). The tests on travis would better be run with both libraries.

**Comment by oscarbenjamin** [comment]
I don't think that MPIR is maintained any more. For example the mpir.org website you've linked to went dead yesterday I think. The two forks of MPIR on GitHub have not seen much recent activity:
https://github.com/wbhart/mpir
https://github.com/BrianGladman/mpir

**Comment by skirpichev** [comment]
Yeah, support for MPIR was removed in 8618bd2.  @casevh, probably this could be closed.
